runtime/rasdump/dmpagent.c

```diff
@@ -655,6 +671,21 @@ doSilentDump(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
 	return OMR_ERROR_NONE;
 }
 
+static omr_error_t
+doJavaVMExit(J9RASdumpAgent *agent, char *label, J9RASdumpContext *context)
+{
+	J9JavaVM *vm = context->javaVM;
+	J9VMThread *vmThread = vm->internalVMFunctions->currentVMThread(vm);
+
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	j9nls_printf(PORTLIB, J9NLS_INFO | J9NLS_STDERR, J9NLS_DMP_EXIT_SHUTDOWN, 
+		context->eventData != NULL ? context->eventData->detailLength : strlen("unknown"), 
+		context->eventData != NULL ? context->eventData->detailData : "unknown");
```

Comment:
I don't think this can be hard coded since it's output as a NLS message. You can add another NLS message for `unknown` (unless there is one already), and retrieve the value at runtime to get the translated version.

Reply:
there are other hard coded nls messages in this file as well https://github.com/eclipse/openj9/blob/a31c051dc099ccb2d6dc6ef3d73fa4d50acc026a/runtime/rasdump/dmpagent.c#L761

Reply:
Are you volunteering to fix them all? :)

Reply:
maybe not at this time.. :D I forgot about the translating aspect.